### ISSUE TITLE
Using a definition-list in references.md doesn't provide link anchors

### ISSUE THREAD

**Issue by wking** [gvwilson/new-lesson-example#5]
For example, you can't currently link to “key word 1” [here][1].  I think it's better to use a subheading for each term to get link anchors, even though that's less semantic.  Requiring users to insert custom HTML to get link anchors seems like a big ask.

If/when kramdown automatically inserts links anchors for definition lists, we can go back to that approach.

[1]: http://gvwilson.github.io/new-lesson-example/reference/

**Comment by gvwilson** [comment]
I agree that manually inserted anchors are a no-go; can we use JQuery to 
insert anchors by scanning the def list and adding them as the page is 
loaded?  (Will linking to an injected anchor of that form actually work?)

**Comment by wking** [comment]
On Mon, Apr 25, 2016 at 02:44:20PM -0700, Greg Wilson wrote:
“… can we use JQuery to insert anchors by scanning the def list and
adding them as the page is loaded?”

Possibly, although I worry about accessibility when using JavaScript
for such a fundamental feature.  I'm trying to decide if I care enough
to file a kramdown patch ;).

**Comment by gvwilson** [comment]
Kramdown's author has been receptive in the past - I just don't know how 
long it'll take a new version to appear on GitHub.

**Comment by wking** [comment]
On Mon, Apr 25, 2016 at 04:52:32PM -0700, Greg Wilson wrote:
“I just don't know how long it'll take a new version to appear on
GitHub.”

It can't take all that many years, and I'm fine playing a long game
here ;).

**Comment by gvwilson** [comment]
Defer until either a kramdown patch appears on GitHub or we decide to patch `<dt>` elements with JS during page load.
